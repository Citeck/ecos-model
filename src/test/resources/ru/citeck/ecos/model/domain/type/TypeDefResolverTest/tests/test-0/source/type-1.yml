---
id: type-1
parentRef: emodel/type@type-0

properties:
  type-1-prop: value-1
