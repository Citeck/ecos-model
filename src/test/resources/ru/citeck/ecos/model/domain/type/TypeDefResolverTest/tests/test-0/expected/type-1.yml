---
id: type-1
name:
  en: type-1

parentRef: emodel/type@type-0
formRef: uiserv/form@form-0
inheritCreateVariants: false

properties:
  type-1-prop: value-1
